custom/icds_reports/tasks.py

```diff
@@ -1741,6 +1753,16 @@ def update_dashboard_activity_report(target_date=None):
         DashboardUserActivityReport().aggregate(target_date)
 
 
+def update_vhnd_form_aggregation(state_id, target_month=None):
```

Comment:
Why this function is created?

Reply:
you asked me to add this just like dashboard activity aggregation. I am using it for testing.Not sure the purpose

Reply:
I see.  well I think function `_agg_gov_vhnd_form` already servers that purpose. So you can remove this.

Reply:
yeah. removed